experimental/Schemes/Sheaves.jl

```diff
@@ -77,7 +77,7 @@ production_func(F::PreSheafOnScheme) = F.production_func
 restriction_func(F::PreSheafOnScheme) = F.restriction_func
 
 ### Production and caching of the values of F on admissible open sets
-function (F::PreSheafOnScheme{<:Any, OpenType, OutputType})(U::T; cached::Bool=true, check::Bool=true) where {OpenType, OutputType, T<:OpenType}
+function (F::PreSheafOnScheme{<:Any, OpenType, OutputType})(U::T; cached::Bool=true, check::Bool=false) where {OpenType, OutputType, T<:OpenType}
```

Comment:
This is a bit delicate. I had the impression that the code would just be running too slow if this was left on `true`. But already, it showed in other discussions that leaving out these sanity checks this immediately leads to difficult bugs. I don't know what's better at the moment.

Reply:
There is a difference between input checks and sanity checks. 
For sanity checks you use `@hassert` with the proper level. Then it is easy to turn them off. 
For input checks of user facing functions there is the `check=true` keyword.